### DISCUSSION TITLE
How to bind to a Task result in code behind?

### DISCUSSION THREAD

**Discussion by dbriard** [AvaloniaUI/Avalonia#15989] [Q&A] [answered]
I read in the doc that you can bind to a task result in XAML with:
```
<TextBlock Text="{Binding MyAsyncText^, FallbackValue='Wait a second'}" />
```
But is it possible to do that in code?

**Comment by maxkatz6** [comment, marked as answer]
You can easily bind to IObservable.
```
textBlock.Bind(TextBlock.TextProperty, observable);
```

In case of Task, you would need to create IObservable wrapper. Or use one from System.Reactive (it had some extensions I think).

**Reply by dbriard** [reply]
Ok, I'll try the wrapper, thank you.
